backend/api/src/main/kotlin/software/shonk/application/service/ShorkService.kt

```diff
@@ -79,9 +79,13 @@ class ShorkService(private val shork: IShork) : ShorkUseCase {
         return Result.failure(IllegalArgumentException(NO_LOBBY_MESSAGE))
     }
 
-    override fun createLobby(): Long {
-        lobbies[lobbyCounter] = Lobby(lobbyCounter, HashMap(), shork)
-        return lobbyCounter++
+    override fun createLobby(playerName: String): Long {
+        if (isAlphaNumerical(playerName)) {
+            lobbies[lobbyCounter] = Lobby(lobbyCounter, HashMap(), shork)
+            lobbies[lobbyCounter]?.joinedPlayers?.add(playerName)
+            return lobbyCounter++
+        }
+        return -1
```

Comment:
As written above, this should not work with -1 as a error code, but with a [Kotlin Result](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/)